Library/ViewModels/CommentRepliesViewModelTests.swift

```diff
@@ -0,0 +1,44 @@
+@testable import KsApi
+@testable import Library
+import Prelude
+import ReactiveExtensions
+import ReactiveExtensions_TestHelpers
+import ReactiveSwift
+import XCTest
+
+internal final class CommentRepliesViewModelTests: TestCase {
+  private let vm: CommentRepliesViewModelType = CommentRepliesViewModel()
+
+  private let loadCommentAndProjectIntoDataSourceComment = TestObserver<Comment, Never>()
+  private let loadCommentAndProjectIntoDataSourceProject = TestObserver<Project, Never>()
+
+  override func setUp() {
+    super.setUp()
+
+    self.vm.outputs.loadCommentAndProjectIntoDataSource.map(first)
+      .observe(self.loadCommentAndProjectIntoDataSourceComment.observer)
+    self.vm.outputs.loadCommentAndProjectIntoDataSource.map(second)
+      .observe(self.loadCommentAndProjectIntoDataSourceProject.observer)
+  }
+
+  func testViewingRootComment_WithCommentAndProject_CommentIsLoadedIntoDataSource() {
```

Comment:
I'm not sure if we have an official rule for this but when I name tests I typically do something like:
`test_{OUTPUTNAME}_{variableX}_{true/false}`

Reply:
Also, the best way to fully test this is to technically test it with the three scenarios:
- just a Project
- just a Comment
- both

Unless I'm misinterpreting the way this works?